### ISSUE TITLE
Reference section is printed twice

### ISSUE THREAD

**Issue by mvuorre** [mvuorre/quarto-preprint#24]
But only on some quarto/typst versions. Waiting for upstream fix?

**Comment by danieltomasz** [comment]
@mvuorre  using Quarto 1.6 release candidate fixes  this for me 
The workaround for quarto 1.5 was adding the following to the yaml (it wasn't supper reliable though)
```
citeproc: false
suppress-bibliography: true # Add issue about this
````
This was a known issue before due to fact how the citation is handled, maybe it is some regression
https://github.com/quarto-dev/quarto-cli/issues/5560

**Comment by mvuorre** [comment]
Righto--this is why I wasn't seeing it on my end. Keeping this open for others to find but won't fix myself.

**Comment by danieltomasz** [comment]
@mvuorre  I did some investigation and found the possible reason you manully invoke citeproc and proces biobliagraphy via pandoc  lua filter in this line https://github.com/mvuorre/quarto-preprint/blob/71bfa84705e9edb9dc95e2913b21696e3fe967be/_extensions/preprint/_extension.yml#L18
after you remove filter, or add an option `suppress-bibliography: true` in the template definition, the bibliography is printed only once and is generated only via typst

**Comment by mvuorre** [comment]
I get it now. Thanks! I've fixed this but hopefully the `suppress-bibliography: true` hack won't be needed in the future.
